control_software/src/snap_fengine.py

```diff
@@ -368,7 +368,7 @@ def get_status(self, jsonify_values=False):
         for stream in range(self.input.ninput_mux_streams):
             status['stream%d_autocorr' % stream] = _cdjsonify(self.corr.get_new_corr(stream, stream).real)
         for stream in range(self.eq.nstreams):
-            for key, val in self.eq.get_status(stream).items():
+            for key, val in self.eq.get_status(stream, include_eq_coeffs).items():
```

Comment:
Suggest keyword assignment here (include_coeffs=include_eq_coeffs) to insulate against position changes.